app/routers/event.py

```diff
@@ -169,6 +170,23 @@ def raise_for_nonexisting_event(event_id: int) -> None:
     )
 
 
+def get_waze_link(event: Event) -> str:
+    """Get a waze navigation link to the event location.
+
+    Returns:
+        If there are coordinates, waze will navigate to the exact location.
+        Otherwise, waze will look for the address that appears in the location.
+        If there is no address, an empty string will be returned."""
+
+    # if event.latitude and event.longitude:
+    #     coordinates = f"{event.latitude},{event.longitude}"
+    #     return f"https://waze.com/ul?ll={coordinates}&navigate=yes"
+    if event.location:
+        url_location = urllib.parse.quote(event.location)
+        return f"https://waze.com/ul?q={url_location}&navigate=yes"
+    return ""
```

Comment:
Prefer to return early (the "guard" pattern):

```
if not event.location:
    return ""
url_location = urllib.parse.quote(event.location)
return f"https://waze.com/ul?q={url_location}&navigate=yes"
```

Less indented and easier to expand later